eng/Signing.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <EnableDefaultArtifacts Condition="'$(DotNetBuild)' != 'true'">false</EnableDefaultArtifacts>
+    <EnableDefaultArtifacts>false</EnableDefaultArtifacts>
```

Comment:
Why can't we use default artifact lookup here anymore and then just update the items and put the `Visibility` metadata on them?

Reply:
By the time we can hook a target in, NuGet packages have already been added to the `ItemsToPushToBlobFeed` item group, which as far as I can tell is not a "documented" repo extension point. We can't manipulate it through the `Artifact` item group. I can likely change this to try to catch it and its symbol package midway through in that group and adjust it.

Reply:
Why do we need a target? Can't we do this at evaluation time?

Reply:
If we do it in evaluation I think we'd end up with duplicate entries as the "default" artifacts globbing for packages doesn't take any efforts to avoid Artifact items.

Reply:
I'm not following. I would expect that we could use the default artifact globbing and then do an item update:

`<Artifact Update="$(ArtifactsPackagesDir)**\Microsoft.NETCore.App.Crossgen2.$(NETCoreSdkRuntimeIdentifier).*.nupkg" Visibility="Vertical" />`

Isn't that possible?

Reply:
The Arcade update is in https://github.com/dotnet/runtime/pull/111220

I have a corresponding change I'll push once that's merged.

Reply:
Updated